rust/transaction_signing/src/sign_transaction.rs

```diff
@@ -59,7 +61,20 @@ pub(crate) fn create_signature(
                 Err(e)
             }
         }
-    }
+    }?;
+
+    let encoded = match &content {
+        SignContent::Transaction {
+            method: _,
+            extensions: _,
+        } => hex::encode(signature.encode()),
+        SignContent::Message(_) => match signature {
+            MultiSignature::Sr25519(a) => hex::encode(a),
+            MultiSignature::Ed25519(a) => hex::encode(a),
+            MultiSignature::Ecdsa(a) => hex::encode(a),
+        },
```

Comment:
* I think that it is not a good pattern to turn a return type from `MultiSignature` to `String`
* Are you sure that this should be just the bare signature that is returned, not `MultiSignature`? How the users are going to tell the difference between different signature schemes? In case of a transaction `MultiSignature` is returned